<!DOCTYPE html>
<html lang="en">
<head>
<meta charset="utf-8"/>
<style>body{background-color:white;}</style>
<link href="common_lib/htmltools-fill-0.5.8.1/fill.css" rel="stylesheet" />
<script src="common_lib/htmlwidgets-1.6.4/htmlwidgets.js"></script>
<link href="common_lib/datatables-css-0.0.0/datatables-crosstalk.css" rel="stylesheet" />
<script src="common_lib/datatables-binding-0.33/datatables.js"></script>
<script src="common_lib/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<link href="common_lib/dt-core-1.13.6/css/jquery.dataTables.min.css" rel="stylesheet" />
<link href="common_lib/dt-core-1.13.6/css/jquery.dataTables.extra.css" rel="stylesheet" />
<script src="common_lib/dt-core-1.13.6/js/jquery.dataTables.min.js"></script>
<script src="common_lib/jszip-1.13.6/jszip.min.js"></script>
<script src="common_lib/pdfmake-1.13.6/pdfmake.js"></script>
<script src="common_lib/pdfmake-1.13.6/vfs_fonts.js"></script>
<link href="common_lib/dt-ext-buttons-1.13.6/css/buttons.dataTables.min.css" rel="stylesheet" />
<script src="common_lib/dt-ext-buttons-1.13.6/js/dataTables.buttons.min.js"></script>
<script src="common_lib/dt-ext-buttons-1.13.6/js/buttons.html5.min.js"></script>
<script src="common_lib/dt-ext-buttons-1.13.6/js/buttons.colVis.min.js"></script>
<script src="common_lib/dt-ext-buttons-1.13.6/js/buttons.print.min.js"></script>
<link href="common_lib/crosstalk-1.2.1/css/crosstalk.min.css" rel="stylesheet" />
<script src="common_lib/crosstalk-1.2.1/js/crosstalk.min.js"></script>
  <title>datatables</title>
</head>
<body>
<div id="htmlwidget_container">
  <div class="datatables html-widget html-fill-item" id="htmlwidget-89c406bd8a1b8b0a1e44" style="width:100%;height:auto;"></div>
</div>
<script type="application/json" data-for="htmlwidget-89c406bd8a1b8b0a1e44">{"x":{"filter":"none","vertical":false,"extensions":["Buttons"],"data":[["1","2","3","4","5","6","7","8","9"],["Multi locus imprinting disorders","Multi locus imprinting disorders","Multi locus imprinting disorders","Multi locus imprinting disorders","Multi locus imprinting disorders","Multi locus imprinting disorders","Multi locus imprinting disorders","Multi locus imprinting disorders","Multi locus imprinting disorders"],["<span class=\"gene-badge\">KHDC3L<\/span>","<span class=\"gene-badge\">NLRP2<\/span>","<span class=\"gene-badge\">NLRP5<\/span>","<span class=\"gene-badge\">NLRP7<\/span>","<span class=\"gene-badge\">PADI6<\/span>","<span class=\"gene-badge\">GRB10<\/span>","<span class=\"gene-badge\">PLAGL1<\/span>","<span class=\"gene-badge\">OOEP<\/span>","<span class=\"gene-badge\">ZFP57<\/span>"],["<span class=\"moi-badge moi-other\">AD/AR<\/span>","<span class=\"moi-badge moi-other\">AD/AR<\/span>","<span class=\"moi-badge moi-other\">AD/AR<\/span>","<span class=\"moi-badge moi-other\">AD/AR<\/span>","<span class=\"moi-badge moi-other\">AD/AR<\/span>","<span class=\"moi-badge moi-ad\">AD<\/span>","<span class=\"moi-badge moi-ad\">AD<\/span>","<span class=\"moi-badge moi-ar\">AR<\/span>","<span class=\"moi-badge moi-ar\">AR<\/span>"],[["Hydatidiform mole, recurrent, 2 OMIM:614293","hydatidiform mole, recurrent, 2 MONDO:0013671"],["Beckwith-Wiedemann syndrome due to imprinting defect of 11p15 MONDO:0016475"],["Phenotype resulting from under expression: Biparental complete hydatidiform mole, Beckwith-Wiedemann Syndrome, Multi-locus imprinting disorder","Affected tissue: all"],["Phenotype resulting from under expression: Biparental complete hydatidiform mole","Affected tissue: all (incompatible with life)","Multi Locus Imprinting Disturbance","hydatidiform mole, recurrent, 1 MONDO:0009273"],["Preimplantation embryonic lethality 2 OMIM:617234","preimplantation embryonic lethality 2 MONDO:0014978","Beckwith-Wiedemann syndrome","Multi Locus Imprinting Disturbance"],["Silver-Russell syndrome 2, OMIM:618905"],["{Diabetes mellitus, transient neonatal 1}\t, OMIM:601410"],["MLID"],["MLID"]],["<a href=\"https://gnomad.broadinstitute.org/gene/KHDC3L\" target=\"_blank\">KHDC3L<\/a>","<a href=\"https://gnomad.broadinstitute.org/gene/NLRP2\" target=\"_blank\">NLRP2<\/a>","<a href=\"https://gnomad.broadinstitute.org/gene/NLRP5\" target=\"_blank\">NLRP5<\/a>","<a href=\"https://gnomad.broadinstitute.org/gene/NLRP7\" target=\"_blank\">NLRP7<\/a>","<a href=\"https://gnomad.broadinstitute.org/gene/PADI6\" target=\"_blank\">PADI6<\/a>","<a href=\"https://gnomad.broadinstitute.org/gene/GRB10\" target=\"_blank\">GRB10<\/a>","<a href=\"https://gnomad.broadinstitute.org/gene/PLAGL1\" target=\"_blank\">PLAGL1<\/a>","<a href=\"https://gnomad.broadinstitute.org/gene/OOEP\" target=\"_blank\">OOEP<\/a>","<a href=\"https://gnomad.broadinstitute.org/gene/ZFP57\" target=\"_blank\">ZFP57<\/a>"],["<a href=\"https://www.genenames.org/data/gene-symbol-report/#!/hgnc_id/HGNC:33699\" target=\"_blank\">HGNC:33699<\/a>","<a href=\"https://www.genenames.org/data/gene-symbol-report/#!/hgnc_id/HGNC:22948\" target=\"_blank\">HGNC:22948<\/a>","<a href=\"https://www.genenames.org/data/gene-symbol-report/#!/hgnc_id/HGNC:21269\" target=\"_blank\">HGNC:21269<\/a>","<a href=\"https://www.genenames.org/data/gene-symbol-report/#!/hgnc_id/HGNC:22947\" target=\"_blank\">HGNC:22947<\/a>","<a href=\"https://www.genenames.org/data/gene-symbol-report/#!/hgnc_id/HGNC:20449\" target=\"_blank\">HGNC:20449<\/a>","<a href=\"https://www.genenames.org/data/gene-symbol-report/#!/hgnc_id/HGNC:4564\" target=\"_blank\">HGNC:4564<\/a>","<a href=\"https://www.genenames.org/data/gene-symbol-report/#!/hgnc_id/HGNC:9046\" target=\"_blank\">HGNC:9046<\/a>","<a href=\"https://www.genenames.org/data/gene-symbol-report/#!/hgnc_id/HGNC:21382\" target=\"_blank\">HGNC:21382<\/a>","<a href=\"https://www.genenames.org/data/gene-symbol-report/#!/hgnc_id/HGNC:18791\" target=\"_blank\">HGNC:18791<\/a>"],["<a href=\"https://www.omim.org/entry/611687\" target=\"_blank\">611687<\/a>","<a href=\"https://www.omim.org/entry/609364\" target=\"_blank\">609364<\/a>","<a href=\"https://www.omim.org/entry/609658\" target=\"_blank\">609658<\/a>","<a href=\"https://www.omim.org/entry/609661\" target=\"_blank\">609661<\/a>","<a href=\"https://www.omim.org/entry/610363\" target=\"_blank\">610363<\/a>","<a href=\"https://www.omim.org/entry/601523\" target=\"_blank\">601523<\/a>","<a href=\"https://www.omim.org/entry/603044\" target=\"_blank\">603044<\/a>","<a href=\"https://www.omim.org/entry/611689\" target=\"_blank\">611689<\/a>","<a href=\"https://www.omim.org/entry/612192\" target=\"_blank\">612192<\/a>"],["6:73362677-73364171","19:54953130-55001142","19:55999726-56061813","19:54923509-54966312","1:17372196-17401699","7:50590063-50793462","6:143940300-144064599","6:73368555-73395133","6:29672392-29681110"],["<a href=\"http://www.ensembl.org/Homo_sapiens/Gene/Summary?db=core;g=ENSG00000203908\" target=\"_blank\">ENSG00000203908<\/a>","<a href=\"http://www.ensembl.org/Homo_sapiens/Gene/Summary?db=core;g=ENSG00000022556\" target=\"_blank\">ENSG00000022556<\/a>","<a href=\"http://www.ensembl.org/Homo_sapiens/Gene/Summary?db=core;g=ENSG00000171487\" target=\"_blank\">ENSG00000171487<\/a>","<a href=\"http://www.ensembl.org/Homo_sapiens/Gene/Summary?db=core;g=ENSG00000167634\" target=\"_blank\">ENSG00000167634<\/a>","<a href=\"http://www.ensembl.org/Homo_sapiens/Gene/Summary?db=core;g=ENSG00000276747\" target=\"_blank\">ENSG00000276747<\/a>","<a href=\"http://www.ensembl.org/Homo_sapiens/Gene/Summary?db=core;g=ENSG00000106070\" target=\"_blank\">ENSG00000106070<\/a>","<a href=\"http://www.ensembl.org/Homo_sapiens/Gene/Summary?db=core;g=ENSG00000118495\" target=\"_blank\">ENSG00000118495<\/a>","<a href=\"http://www.ensembl.org/Homo_sapiens/Gene/Summary?db=core;g=ENSG00000203907\" target=\"_blank\">ENSG00000203907<\/a>","<a href=\"http://www.ensembl.org/Homo_sapiens/Gene/Summary?db=core;g=ENSG00000204644\" target=\"_blank\">ENSG00000204644<\/a>"],["3","3","3","3","3","2","2","0","0"],["Complete","Complete","Complete","Complete",null,null,null,"unknown","unknown"],[["ECAT1"],["FLJ20510","PYPAF2","NBS1","PAN1","CLR19.9"],["PYPAF8","MATER","PAN11","CLR19.8"],["PYPAF3","NOD12","PAN7","CLR19.4"],[],[],["ZAC","LOT1"],["Em:AC019205.2","KHDC2"],["ZNF698","bA145L22","bA145L22.2"]],["KH domain containing 3 like, subcortical maternal complex member","NLR family pyrin domain containing 2","NLR family pyrin domain containing 5","NLR family pyrin domain containing 7","peptidyl arginine deiminase 6","growth factor receptor bound protein 10","PLAG1 like zinc finger 1","oocyte expressed protein","ZFP57 zinc finger protein"],["ES cell associated transcript 1","nucleotide-binding oligomerization domain, leucine rich repeat and pyrin domain containing 2",["nucleotide-binding oligomerization domain, leucine rich repeat and pyrin domain containing 5","maternal antigen that embryos require"],"nucleotide-binding oligomerization domain, leucine rich repeat and pyrin domain containing 7",null,null,null,"KH homology domain containing 2",null],["KHDC3L","NLRP2","NLRP5","NLRP7","PADI6","GRB10","PLAGL1","OOEP","ZFP57"],[1109,1109,1109,1109,1109,1109,1109,1109,1109],["Endocrinology","Endocrinology","Endocrinology","Endocrinology","Endocrinology","Endocrinology","Endocrinology","Endocrinology","Endocrinology"],["","","","","","","","",""],["BOTH monoallelic and biallelic (but BIALLELIC mutations cause a more SEVERE disease form), autosomal or pseudoautosomal","BOTH monoallelic and biallelic (but BIALLELIC mutations cause a more SEVERE disease form), autosomal or pseudoautosomal","BOTH monoallelic and biallelic (but BIALLELIC mutations cause a more SEVERE disease form), autosomal or pseudoautosomal","BOTH monoallelic and biallelic (but BIALLELIC mutations cause a more SEVERE disease form), autosomal or pseudoautosomal","BOTH monoallelic and biallelic (but BIALLELIC mutations cause a more SEVERE disease form), autosomal or pseudoautosomal","MONOALLELIC, autosomal or pseudoautosomal, paternally imprinted (maternal allele expressed)","MONOALLELIC, autosomal or pseudoautosomal, maternally imprinted (paternal allele expressed)","BIALLELIC, autosomal or pseudoautosomal","BIALLELIC, autosomal or pseudoautosomal"],[[],["watchlist","gene-checked"],["gene-checked"],[],[],[],[],[],[]],["6:74072400-74073894","19:55464498-55512510","19:56511092-56573179","19:55434877-55477680","1:17698691-17728195","7:50657760-50861159","6:144261437-144385735","6:74078278-74104856","6:29640169-29648887"],["ENSG00000203908","ENSG00000022556","ENSG00000171487","ENSG00000167634","ENSG00000256049","ENSG00000106070","ENSG00000118495","ENSG00000203907","ENSG00000204644"],[["21885028","23232697","31847873","31201414"],["19300480","30221575","32169557","28422141","28317850","26323243","29574422","30877238","33090377"],["26323243","31201414","31829238"],["16462743","19246479","31201414","29574422","28916717"],["32928291","33221824","27545678"],["10861285","10856193","27370225","10856193","11112662","30794780","http://igc.otago.ac.nz/home.html"],["16928428","7719335","8842729","11935319","17084362","10655556","30794780","http://igc.otago.ac.nz/home.html"],["PMID: 39090763"],["PMID: 39090763"]],[1109,1109,1109,1109,1109,1109,1109,1109,1109]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>name<\/th>\n      <th>entity name<\/th>\n      <th>mode of inheritance<\/th>\n      <th>phenotypes<\/th>\n      <th>hgnc symbol gnomad<\/th>\n      <th>gene data hgnc id<\/th>\n      <th>gene data omim gene<\/th>\n      <th>gene data ensembl genes GRch38 90 location<\/th>\n      <th>gene data ensembl genes GRch38 90 ensembl id<\/th>\n      <th>confidence level<\/th>\n      <th>penetrance<\/th>\n      <th>gene data alias<\/th>\n      <th>gene data gene name<\/th>\n      <th>gene data alias name<\/th>\n      <th>gene data gene symbol<\/th>\n      <th>id<\/th>\n      <th>disease group<\/th>\n      <th>disease sub group<\/th>\n      <th>mode of inheritance raw<\/th>\n      <th>tags<\/th>\n      <th>gene data ensembl genes GRch37 82 location<\/th>\n      <th>gene data ensembl genes GRch37 82 ensembl id<\/th>\n      <th>publications<\/th>\n      <th>panel id<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"dom":"Bfrtip","buttons":["copy","csv","excel","pdf","print"],"pageLength":20,"language":{"search":"","searchPlaceholder":"Enter natural language query..."},"initComplete":"\n        function(settings, json) {\n\n          var filter = $('div.dataTables_filter');\n          filter.css({'width': '100%'});\n          filter.find('input').css({\n            'width': '100%',\n            'border': '1px solid #ccc',\n            'padding': '8px',\n            'border-radius': '4px',\n            'box-shadow': '0 1px 3px rgba(0,0,0,0.2)',\n            'font-size': '14px',\n            'font-family': 'system-ui, -apple-system, BlinkMacSystemFont, \"Segoe UI\", Roboto, Oxygen, Ubuntu, Cantarell, \"Open Sans\", \"Helvetica Neue\", sans-serif'\n          });\n\n          $('table.dataTable').css({\n            'font-family': 'system-ui, -apple-system, BlinkMacSystemFont, \"Segoe UI\", Roboto, Oxygen, Ubuntu, Cantarell, \"Open Sans\", \"Helvetica Neue\", sans-serif'\n          });\n\n          if (!document.getElementById('moi-badge-style')) {\n            var st = document.createElement('style');\n            st.id = 'moi-badge-style';\n            st.type = 'text/css';\n            st.innerHTML = `\n              .moi-badge {\n                display: inline-block;\n                padding: 4px 10px;\n                margin: 2px 4px 2px 0;\n                border-radius: 14px;\n                font-size: 13px;\n                font-weight: 600;\n                letter-spacing: 0.2px;\n                line-height: 1.2;\n                color: #ffffff;\n                font-family: system-ui, -apple-system, BlinkMacSystemFont, 'Segoe UI',\n                             Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans',\n                             'Helvetica Neue', sans-serif;\n              }\n              .gene-badge {\n  display: inline-block;\n  padding: 4px 10px;\n  margin: 2px 4px 2px 0;\n  border-radius: 14px;\n  font-size: 13px;\n  font-weight: 600;\n  letter-spacing: 0.2px;\n  line-height: 1.2;\n  background: #e3f2fd;\n  color: #0d47a1;\n  border: 1px solid rgba(13,71,161,0.15);\n  font-family: system-ui, -apple-system, BlinkMacSystemFont, 'Segoe UI',\n               Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans',\n               'Helvetica Neue', sans-serif;\n}\n              .moi-ar { background: #2e7d32; }\n              .moi-ad { background: #1565c0; }\n              .moi-xl { background: #6a1b9a; }\n              .moi-mt { background: #ef6c00; }\n              .moi-other { background: #546e7a; }\n            `;\n            document.head.appendChild(st);\n          }\n\n        }\n      ","columnDefs":[{"className":"dt-right","targets":[16,24]},{"orderable":false,"targets":0},{"name":" ","targets":0},{"name":"name","targets":1},{"name":"entity name","targets":2},{"name":"mode of inheritance","targets":3},{"name":"phenotypes","targets":4},{"name":"hgnc symbol gnomad","targets":5},{"name":"gene data hgnc id","targets":6},{"name":"gene data omim gene","targets":7},{"name":"gene data ensembl genes GRch38 90 location","targets":8},{"name":"gene data ensembl genes GRch38 90 ensembl id","targets":9},{"name":"confidence level","targets":10},{"name":"penetrance","targets":11},{"name":"gene data alias","targets":12},{"name":"gene data gene name","targets":13},{"name":"gene data alias name","targets":14},{"name":"gene data gene symbol","targets":15},{"name":"id","targets":16},{"name":"disease group","targets":17},{"name":"disease sub group","targets":18},{"name":"mode of inheritance raw","targets":19},{"name":"tags","targets":20},{"name":"gene data ensembl genes GRch37 82 location","targets":21},{"name":"gene data ensembl genes GRch37 82 ensembl id","targets":22},{"name":"publications","targets":23},{"name":"panel id","targets":24}],"order":[],"autoWidth":false,"orderClasses":false,"lengthMenu":[10,20,25,50,100]}},"evals":["options.initComplete"],"jsHooks":[]}</script>
<script type="application/htmlwidget-sizing" data-for="htmlwidget-89c406bd8a1b8b0a1e44">{"viewer":{"width":"100%","height":"auto","padding":15,"fill":true},"browser":{"width":"100%","height":"auto","padding":40,"fill":false}}</script>
</body>
</html>
