<!DOCTYPE html>
<html lang="en">
<head>
<meta charset="utf-8"/>
<style>body{background-color:white;}</style>
<link href="common_lib/htmltools-fill-0.5.8.1/fill.css" rel="stylesheet" />
<script src="common_lib/htmlwidgets-1.6.4/htmlwidgets.js"></script>
<link href="common_lib/datatables-css-0.0.0/datatables-crosstalk.css" rel="stylesheet" />
<script src="common_lib/datatables-binding-0.33/datatables.js"></script>
<script src="common_lib/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<link href="common_lib/dt-core-1.13.6/css/jquery.dataTables.min.css" rel="stylesheet" />
<link href="common_lib/dt-core-1.13.6/css/jquery.dataTables.extra.css" rel="stylesheet" />
<script src="common_lib/dt-core-1.13.6/js/jquery.dataTables.min.js"></script>
<script src="common_lib/jszip-1.13.6/jszip.min.js"></script>
<script src="common_lib/pdfmake-1.13.6/pdfmake.js"></script>
<script src="common_lib/pdfmake-1.13.6/vfs_fonts.js"></script>
<link href="common_lib/dt-ext-buttons-1.13.6/css/buttons.dataTables.min.css" rel="stylesheet" />
<script src="common_lib/dt-ext-buttons-1.13.6/js/dataTables.buttons.min.js"></script>
<script src="common_lib/dt-ext-buttons-1.13.6/js/buttons.html5.min.js"></script>
<script src="common_lib/dt-ext-buttons-1.13.6/js/buttons.colVis.min.js"></script>
<script src="common_lib/dt-ext-buttons-1.13.6/js/buttons.print.min.js"></script>
<link href="common_lib/crosstalk-1.2.1/css/crosstalk.min.css" rel="stylesheet" />
<script src="common_lib/crosstalk-1.2.1/js/crosstalk.min.js"></script>
  <title>datatables</title>
</head>
<body>
<div id="htmlwidget_container">
  <div class="datatables html-widget html-fill-item" id="htmlwidget-4861213c3c1c18ae6322" style="width:100%;height:auto;"></div>
</div>
<script type="application/json" data-for="htmlwidget-4861213c3c1c18ae6322">{"x":{"filter":"none","vertical":false,"extensions":["Buttons"],"data":[["1","2","3","4","5","6","7","8","9","10","11","12","13","14"],["Familial prostate cancer","Familial prostate cancer","Familial prostate cancer","Familial prostate cancer","Familial prostate cancer","Familial prostate cancer","Familial prostate cancer","Familial prostate cancer","Familial prostate cancer","Familial prostate cancer","Familial prostate cancer","Familial prostate cancer","Familial prostate cancer","Familial prostate cancer"],["<span class=\"gene-badge\">BRCA2<\/span>","<span class=\"gene-badge\">BRCA1<\/span>","<span class=\"gene-badge\">BRIP1<\/span>","<span class=\"gene-badge\">CHEK2<\/span>","<span class=\"gene-badge\">ELAC2<\/span>","<span class=\"gene-badge\">HOXB13<\/span>","<span class=\"gene-badge\">MSH2<\/span>","<span class=\"gene-badge\">MSH5<\/span>","<span class=\"gene-badge\">MSH6<\/span>","<span class=\"gene-badge\">MSR1<\/span>","<span class=\"gene-badge\">PALB2<\/span>","<span class=\"gene-badge\">PTEN<\/span>","<span class=\"gene-badge\">RNASEL<\/span>","<span class=\"gene-badge\">SPOP<\/span>"],["<span class=\"moi-badge moi-ad\">AD<\/span>","<span class=\"moi-badge moi-ad\">AD<\/span>","","<span class=\"moi-badge moi-ad\">AD<\/span>","","","","","","<span class=\"moi-badge moi-ad\">AD<\/span>","","<span class=\"moi-badge moi-ar\">AR<\/span>","","<span class=\"moi-badge moi-ar\">AR<\/span>"],[["{Prostate cancer}, OMIM:176807"],["Prostate cancer, MONDO:0008315"],[],["{Prostate cancer, familial, susceptibility to}"],["hereditary prostate cancer"],[],[],["prostate cancer susceptibility"],[],["Prostate cancer, hereditary"],[],["harmartoma, glioma, prostate, endometrial","Cowden syndrome, Bannayan-Riley-Ruvalcaba syndrome"],["hereditary prostate cancer"],["prostate cancer"]],["<a href=\"https://gnomad.broadinstitute.org/gene/BRCA2\" target=\"_blank\">BRCA2<\/a>","<a href=\"https://gnomad.broadinstitute.org/gene/BRCA1\" target=\"_blank\">BRCA1<\/a>","<a href=\"https://gnomad.broadinstitute.org/gene/BRIP1\" target=\"_blank\">BRIP1<\/a>","<a href=\"https://gnomad.broadinstitute.org/gene/CHEK2\" target=\"_blank\">CHEK2<\/a>","<a href=\"https://gnomad.broadinstitute.org/gene/ELAC2\" target=\"_blank\">ELAC2<\/a>","<a href=\"https://gnomad.broadinstitute.org/gene/HOXB13\" target=\"_blank\">HOXB13<\/a>","<a href=\"https://gnomad.broadinstitute.org/gene/MSH2\" target=\"_blank\">MSH2<\/a>","<a href=\"https://gnomad.broadinstitute.org/gene/MSH5\" target=\"_blank\">MSH5<\/a>","<a href=\"https://gnomad.broadinstitute.org/gene/MSH6\" target=\"_blank\">MSH6<\/a>","<a href=\"https://gnomad.broadinstitute.org/gene/MSR1\" target=\"_blank\">MSR1<\/a>","<a href=\"https://gnomad.broadinstitute.org/gene/PALB2\" target=\"_blank\">PALB2<\/a>","<a href=\"https://gnomad.broadinstitute.org/gene/PTEN\" target=\"_blank\">PTEN<\/a>","<a href=\"https://gnomad.broadinstitute.org/gene/RNASEL\" target=\"_blank\">RNASEL<\/a>","<a href=\"https://gnomad.broadinstitute.org/gene/SPOP\" target=\"_blank\">SPOP<\/a>"],["<a href=\"https://www.genenames.org/data/gene-symbol-report/#!/hgnc_id/HGNC:1101\" target=\"_blank\">HGNC:1101<\/a>","<a href=\"https://www.genenames.org/data/gene-symbol-report/#!/hgnc_id/HGNC:1100\" target=\"_blank\">HGNC:1100<\/a>","<a href=\"https://www.genenames.org/data/gene-symbol-report/#!/hgnc_id/HGNC:20473\" target=\"_blank\">HGNC:20473<\/a>","<a href=\"https://www.genenames.org/data/gene-symbol-report/#!/hgnc_id/HGNC:16627\" target=\"_blank\">HGNC:16627<\/a>","<a href=\"https://www.genenames.org/data/gene-symbol-report/#!/hgnc_id/HGNC:14198\" target=\"_blank\">HGNC:14198<\/a>","<a href=\"https://www.genenames.org/data/gene-symbol-report/#!/hgnc_id/HGNC:5112\" target=\"_blank\">HGNC:5112<\/a>","<a href=\"https://www.genenames.org/data/gene-symbol-report/#!/hgnc_id/HGNC:7325\" target=\"_blank\">HGNC:7325<\/a>","<a href=\"https://www.genenames.org/data/gene-symbol-report/#!/hgnc_id/HGNC:7328\" target=\"_blank\">HGNC:7328<\/a>","<a href=\"https://www.genenames.org/data/gene-symbol-report/#!/hgnc_id/HGNC:7329\" target=\"_blank\">HGNC:7329<\/a>","<a href=\"https://www.genenames.org/data/gene-symbol-report/#!/hgnc_id/HGNC:7376\" target=\"_blank\">HGNC:7376<\/a>","<a href=\"https://www.genenames.org/data/gene-symbol-report/#!/hgnc_id/HGNC:26144\" target=\"_blank\">HGNC:26144<\/a>","<a href=\"https://www.genenames.org/data/gene-symbol-report/#!/hgnc_id/HGNC:9588\" target=\"_blank\">HGNC:9588<\/a>","<a href=\"https://www.genenames.org/data/gene-symbol-report/#!/hgnc_id/HGNC:10050\" target=\"_blank\">HGNC:10050<\/a>","<a href=\"https://www.genenames.org/data/gene-symbol-report/#!/hgnc_id/HGNC:11254\" target=\"_blank\">HGNC:11254<\/a>"],["<a href=\"https://www.omim.org/entry/600185\" target=\"_blank\">600185<\/a>","<a href=\"https://www.omim.org/entry/113705\" target=\"_blank\">113705<\/a>","<a href=\"https://www.omim.org/entry/605882\" target=\"_blank\">605882<\/a>","<a href=\"https://www.omim.org/entry/604373\" target=\"_blank\">604373<\/a>","<a href=\"https://www.omim.org/entry/605367\" target=\"_blank\">605367<\/a>","<a href=\"https://www.omim.org/entry/604607\" target=\"_blank\">604607<\/a>","<a href=\"https://www.omim.org/entry/609309\" target=\"_blank\">609309<\/a>","<a href=\"https://www.omim.org/entry/603382\" target=\"_blank\">603382<\/a>","<a href=\"https://www.omim.org/entry/600678\" target=\"_blank\">600678<\/a>","<a href=\"https://www.omim.org/entry/153622\" target=\"_blank\">153622<\/a>","<a href=\"https://www.omim.org/entry/610355\" target=\"_blank\">610355<\/a>","<a href=\"https://www.omim.org/entry/601728\" target=\"_blank\">601728<\/a>","<a href=\"https://www.omim.org/entry/180435\" target=\"_blank\">180435<\/a>","<a href=\"https://www.omim.org/entry/602650\" target=\"_blank\">602650<\/a>"],["13:32315474-32400266","17:43044295-43170245","17:61681266-61863521","22:28687743-28742422","17:12992391-13018187","17:48724763-48729178","2:47402969-47562311","6:31739948-31762834","2:47695530-47810101","8:16107878-16567490","16:23603160-23641310","10:87863113-87971930","1:182573634-182589256","17:49598884-49678234"],["<a href=\"http://www.ensembl.org/Homo_sapiens/Gene/Summary?db=core;g=ENSG00000139618\" target=\"_blank\">ENSG00000139618<\/a>","<a href=\"http://www.ensembl.org/Homo_sapiens/Gene/Summary?db=core;g=ENSG00000012048\" target=\"_blank\">ENSG00000012048<\/a>","<a href=\"http://www.ensembl.org/Homo_sapiens/Gene/Summary?db=core;g=ENSG00000136492\" target=\"_blank\">ENSG00000136492<\/a>","<a href=\"http://www.ensembl.org/Homo_sapiens/Gene/Summary?db=core;g=ENSG00000183765\" target=\"_blank\">ENSG00000183765<\/a>","<a href=\"http://www.ensembl.org/Homo_sapiens/Gene/Summary?db=core;g=ENSG00000006744\" target=\"_blank\">ENSG00000006744<\/a>","<a href=\"http://www.ensembl.org/Homo_sapiens/Gene/Summary?db=core;g=ENSG00000159184\" target=\"_blank\">ENSG00000159184<\/a>","<a href=\"http://www.ensembl.org/Homo_sapiens/Gene/Summary?db=core;g=ENSG00000095002\" target=\"_blank\">ENSG00000095002<\/a>","<a href=\"http://www.ensembl.org/Homo_sapiens/Gene/Summary?db=core;g=ENSG00000204410\" target=\"_blank\">ENSG00000204410<\/a>","<a href=\"http://www.ensembl.org/Homo_sapiens/Gene/Summary?db=core;g=ENSG00000116062\" target=\"_blank\">ENSG00000116062<\/a>","<a href=\"http://www.ensembl.org/Homo_sapiens/Gene/Summary?db=core;g=ENSG00000038945\" target=\"_blank\">ENSG00000038945<\/a>","<a href=\"http://www.ensembl.org/Homo_sapiens/Gene/Summary?db=core;g=ENSG00000083093\" target=\"_blank\">ENSG00000083093<\/a>","<a href=\"http://www.ensembl.org/Homo_sapiens/Gene/Summary?db=core;g=ENSG00000171862\" target=\"_blank\">ENSG00000171862<\/a>","<a href=\"http://www.ensembl.org/Homo_sapiens/Gene/Summary?db=core;g=ENSG00000135828\" target=\"_blank\">ENSG00000135828<\/a>","<a href=\"http://www.ensembl.org/Homo_sapiens/Gene/Summary?db=core;g=ENSG00000121067\" target=\"_blank\">ENSG00000121067<\/a>"],["3","2","1","1","1","1","1","1","1","1","1","1","1","1"],["Complete","Complete","Complete","Complete","Complete","Complete","Complete","Complete","Complete","Complete","Complete","Complete","Complete","Complete"],[["FAD","FAD1","BRCC2","XRCC11"],["RNF53","BRCC1","PPP1R53","FANCS"],["OF","BACH1","FANCJ"],["CDS1","CHK2","HuCds1","PP1425","bA444G7"],["FLJ10530","HPC2"],[],["HNPCC","HNPCC1"],["G7"],[],["SCARA1","CD204","SR-AI","SR-AII","SR-AIII","SR-A"],["FLJ21816","FANCN"],["MMAC1","TEP1","PTEN1"],[],["TEF2","BTBD32"]],["BRCA2, DNA repair associated","BRCA1, DNA repair associated","BRCA1 interacting protein C-terminal helicase 1","checkpoint kinase 2","elaC ribonuclease Z 2","homeobox B13","mutS homolog 2","mutS homolog 5","mutS homolog 6","macrophage scavenger receptor 1","partner and localizer of BRCA2","phosphatase and tensin homolog","ribonuclease L","speckle type BTB/POZ protein"],["BRCA1/BRCA2-containing complex, subunit 2",["BRCA1/BRCA2-containing complex, subunit 1","protein phosphatase 1, regulatory subunit 53","Fanconi anemia, complementation group S"],"BRCA1/BRCA2-associated helicase 1",null,"tRNase Z (long form)",null,null,null,null,null,"Fanconi anemia, complementation group N","mutated in multiple advanced cancers 1",null,null],["BRCA2","BRCA1","BRIP1","CHEK2","ELAC2","HOXB13","MSH2","MSH5","MSH6","MSR1","PALB2","PTEN","RNASEL","SPOP"],[318,318,318,318,318,318,318,318,318,318,318,318,318,318],["","","","","","","","","","","","","",""],["","","","","","","","","","","","","",""],["MONOALLELIC, autosomal or pseudoautosomal, NOT imprinted","MONOALLELIC, autosomal or pseudoautosomal, NOT imprinted","Unknown","MONOALLELIC, autosomal or pseudoautosomal, imprinted status unknown","Unknown","Unknown","Unknown","Unknown","Unknown","MONOALLELIC, autosomal or pseudoautosomal, NOT imprinted","Unknown","BIALLELIC, autosomal or pseudoautosomal","Unknown","BIALLELIC, autosomal or pseudoautosomal"],[[],[],[],[],[],[],[],[],[],[],[],[],[],[]],["13:32889611-32973805","17:41196312-41277500","17:59758627-59940882","22:29083731-29138410","17:12895708-12921504","17:46802125-46806540","2:47630108-47789450","6:31707725-31732622","2:47922669-48037240","8:15965387-16424999","16:23614488-23652631","10:89622870-89731687","1:182542769-182558391","17:47676246-47755596"],["ENSG00000139618","ENSG00000012048","ENSG00000136492","ENSG00000183765","ENSG00000006744","ENSG00000159184","ENSG00000095002","ENSG00000204410","ENSG00000116062","ENSG00000038945","ENSG00000083093","ENSG00000171862","ENSG00000135828","ENSG00000121067"],[["26360800","10398279 - no association found with breast/ovarian cance susceptibility variants","26289772","26236408 - literature review indicating there is conflicting reports","26074382","21952622","28031937"],["27989354","27899188","27742670","27433846","27317574","27306910 - standardized incidence ratios (SIRs) were used to compare the observed incidence of 20 primary cancer sites to the expected incidence of each cancer based on the calculated risk estimates according to each subject's age, sex, and ethnicity. BRCA1 families had increased SIRs for breast and ovarian cancer (p < .001) and decreased SIRs for kidney, lung, prostate, and thyroid cancer and non-Hodgkin's lymphoma (p < .001)","27188668","26926928","26360800 - no increased risk","26236408 - literature review indicating there is conflicting reports","22516946","28031937"],["26074382"],["26074382"],["28031937"],["26074382","28031937"],["26289772"],["26964030"],["27456091"],["12244320","12244320","12958598 - found no association and the variant reported in PMID:12244320 has been reclassified","27084275","26225686 - no association with another variant found in meta-analysis","28031937"],["26074382"],[],["28031937"],[]],[318,318,318,318,318,318,318,318,318,318,318,318,318,318]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>name<\/th>\n      <th>entity name<\/th>\n      <th>mode of inheritance<\/th>\n      <th>phenotypes<\/th>\n      <th>hgnc symbol gnomad<\/th>\n      <th>gene data hgnc id<\/th>\n      <th>gene data omim gene<\/th>\n      <th>gene data ensembl genes GRch38 90 location<\/th>\n      <th>gene data ensembl genes GRch38 90 ensembl id<\/th>\n      <th>confidence level<\/th>\n      <th>penetrance<\/th>\n      <th>gene data alias<\/th>\n      <th>gene data gene name<\/th>\n      <th>gene data alias name<\/th>\n      <th>gene data gene symbol<\/th>\n      <th>id<\/th>\n      <th>disease group<\/th>\n      <th>disease sub group<\/th>\n      <th>mode of inheritance raw<\/th>\n      <th>tags<\/th>\n      <th>gene data ensembl genes GRch37 82 location<\/th>\n      <th>gene data ensembl genes GRch37 82 ensembl id<\/th>\n      <th>publications<\/th>\n      <th>panel id<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"dom":"Bfrtip","buttons":["copy","csv","excel","pdf","print"],"pageLength":20,"language":{"search":"","searchPlaceholder":"Enter natural language query..."},"initComplete":"\n        function(settings, json) {\n\n          var filter = $('div.dataTables_filter');\n          filter.css({'width': '100%'});\n          filter.find('input').css({\n            'width': '100%',\n            'border': '1px solid #ccc',\n            'padding': '8px',\n            'border-radius': '4px',\n            'box-shadow': '0 1px 3px rgba(0,0,0,0.2)',\n            'font-size': '14px',\n            'font-family': 'system-ui, -apple-system, BlinkMacSystemFont, \"Segoe UI\", Roboto, Oxygen, Ubuntu, Cantarell, \"Open Sans\", \"Helvetica Neue\", sans-serif'\n          });\n\n          $('table.dataTable').css({\n            'font-family': 'system-ui, -apple-system, BlinkMacSystemFont, \"Segoe UI\", Roboto, Oxygen, Ubuntu, Cantarell, \"Open Sans\", \"Helvetica Neue\", sans-serif'\n          });\n\n          if (!document.getElementById('moi-badge-style')) {\n            var st = document.createElement('style');\n            st.id = 'moi-badge-style';\n            st.type = 'text/css';\n            st.innerHTML = `\n              .moi-badge {\n                display: inline-block;\n                padding: 4px 10px;\n                margin: 2px 4px 2px 0;\n                border-radius: 14px;\n                font-size: 13px;\n                font-weight: 600;\n                letter-spacing: 0.2px;\n                line-height: 1.2;\n                color: #ffffff;\n                font-family: system-ui, -apple-system, BlinkMacSystemFont, 'Segoe UI',\n                             Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans',\n                             'Helvetica Neue', sans-serif;\n              }\n              .gene-badge {\n  display: inline-block;\n  padding: 4px 10px;\n  margin: 2px 4px 2px 0;\n  border-radius: 14px;\n  font-size: 13px;\n  font-weight: 600;\n  letter-spacing: 0.2px;\n  line-height: 1.2;\n  background: #e3f2fd;\n  color: #0d47a1;\n  border: 1px solid rgba(13,71,161,0.15);\n  font-family: system-ui, -apple-system, BlinkMacSystemFont, 'Segoe UI',\n               Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans',\n               'Helvetica Neue', sans-serif;\n}\n              .moi-ar { background: #2e7d32; }\n              .moi-ad { background: #1565c0; }\n              .moi-xl { background: #6a1b9a; }\n              .moi-mt { background: #ef6c00; }\n              .moi-other { background: #546e7a; }\n            `;\n            document.head.appendChild(st);\n          }\n\n        }\n      ","columnDefs":[{"className":"dt-right","targets":[16,24]},{"orderable":false,"targets":0},{"name":" ","targets":0},{"name":"name","targets":1},{"name":"entity name","targets":2},{"name":"mode of inheritance","targets":3},{"name":"phenotypes","targets":4},{"name":"hgnc symbol gnomad","targets":5},{"name":"gene data hgnc id","targets":6},{"name":"gene data omim gene","targets":7},{"name":"gene data ensembl genes GRch38 90 location","targets":8},{"name":"gene data ensembl genes GRch38 90 ensembl id","targets":9},{"name":"confidence level","targets":10},{"name":"penetrance","targets":11},{"name":"gene data alias","targets":12},{"name":"gene data gene name","targets":13},{"name":"gene data alias name","targets":14},{"name":"gene data gene symbol","targets":15},{"name":"id","targets":16},{"name":"disease group","targets":17},{"name":"disease sub group","targets":18},{"name":"mode of inheritance raw","targets":19},{"name":"tags","targets":20},{"name":"gene data ensembl genes GRch37 82 location","targets":21},{"name":"gene data ensembl genes GRch37 82 ensembl id","targets":22},{"name":"publications","targets":23},{"name":"panel id","targets":24}],"order":[],"autoWidth":false,"orderClasses":false,"lengthMenu":[10,20,25,50,100]}},"evals":["options.initComplete"],"jsHooks":[]}</script>
<script type="application/htmlwidget-sizing" data-for="htmlwidget-4861213c3c1c18ae6322">{"viewer":{"width":"100%","height":"auto","padding":15,"fill":true},"browser":{"width":"100%","height":"auto","padding":40,"fill":false}}</script>
</body>
</html>
